pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "pathfinder-monster-creator"
+version = "0.1.0"
+description = "Project supporting pen & paper RPG game design using machine learning."
+authors = [
+    "Paulina Jędrychowska <>",
+    "Bogumiła Papiernik <>",
+    "Jolanta Śliwa <>"
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+pandas = "1.4.*"
+numpy = "1.21.*"
+scikit-learn = "1.2.*"
+
+[tool.poetry.group.style.dependencies]
+pre-commit = "2.20.*"
```

Comment:
Tutaj też można dodać `black`, jakby ktoś chciał go odpalić ręcznie, a nie tylko automatycznie przed commitem. Ale to trochę opcjonalne